v4-client-py/v4_client_py/clients/helpers/chain_helpers.py

```diff
@@ -25,6 +26,41 @@ class OrderExecution(Flag):
     POST_ONLY = auto()  # Post-only
     FOK = auto()        # Fill or Kill
 
+# Enums to use in order proto fields. Use proto generated fields once that's fixed.
+class Order_TimeInForce(Flag):
+
+    '''
+    TIME_IN_FORCE_UNSPECIFIED - TIME_IN_FORCE_UNSPECIFIED represents the default behavior where an
+    order will first match with existing orders on the book, and any
+    remaining size will be added to the book as a maker order.
+    '''
+    TIME_IN_FORCE_UNSPECIFIED = 0
+
+    '''
+    TIME_IN_FORCE_IOC - TIME_IN_FORCE_IOC enforces that an order only be matched with
+    maker orders on the book. If the order has remaining size after
+    matching with existing orders on the book, the remaining size
+    is not placed on the book.
+    '''
+    TIME_IN_FORCE_IOC = 1
```

Comment:
Up to you, but personally I'm fine with using `auto()` here as long as you specify the start.